target/linux/ramips/dts/mt7628an_asus_rt-ac1200.dts

```diff
@@ -0,0 +1,183 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+// Copyright (c) 2022 Ray Wang
+// Copyright (c) 2022 Ivan Pavlov
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "asus,rt-ac1200", "mediatek,mt7628an-soc";
+	model = "Asus RT-AC1200";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 5 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "blue:power";
+			gpios = <&gpio 37 GPIO_ACTIVE_LOW>;
+			default-state = "keep";
+		};
+		
+		all {
+			label = "blue:all";
```

Comment:
Is it an actual LED or a gpio line which turns on/off all the other LEDs?

Reply:
This GPIO is the master switch to turn on/off all LEDs except for the power LED

Reply:
Thanks, I'll look into it.

Reply:
```
	gpio_export {
		compatible = "gpio-export";
		#size-cells = <0>;

		led_all {
			gpio-export,name = "led_all";
			gpio-export,output = <0>;
			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
		};
	};
```
Ended up declaring it this way, because if not defined it will default to disabling all LEDs, and it would be ideal to have LEDs during boot.